app/src/main/kotlin/com/wire/android/ui/home/conversations/details/editguestaccess/EditGuestAccessViewModel.kt

```diff
@@ -224,19 +224,19 @@ class EditGuestAccessViewModel @Inject constructor(
     }
 
     fun onRevokeDialogConfirm() {
-        removeGuestLink()
+        viewModelScope.launch {
+            removeGuestLink()
```

Comment:
should we move this the kalium usecase 

1. revoke guests access
2. on success revoke guest link

Reply:
Ok, but revoke should be before revoking guest access